core/src/main/resources/hudson/model/UpdateCenter/update-center.js

```diff
@@ -14,33 +14,38 @@ Behaviour.specify(
 
 function refresh() {
   window.setTimeout(function () {
-    new Ajax.Request("./body", {
-      onSuccess: function (rsp) {
-        var div = document.createElement("div");
-        div.innerHTML = rsp.responseText;
+    fetch("./body").then((rsp) => {
+      {
```

Comment:
[nit] Seems like this extra level of braces is unnecessary and could be removed to decrease the level of indentation.